### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by atatmaja**
## Briefly describe the feature or bug that this PR addresses.

Creates a 404 page that links users back to the homepage (Explore) 

## Please link the Airtable ticket associated with this PR.

https://airtable.com/tbli2lWQHAqBa6ZcI/viwWvxb27XGuziw4c/recCfFJ1JWnzzIU2o

## If applicable, include screenshots of the feature/bugfix introduced by this PR (Please include both desktop and mobile if there is a significant UI change).

image

image

## Describe the steps you took to test the feature/bugfix introduced by this PR.

Manual smoke testing.

## Does this PR depend on any recent backend work? If so, please include the PR number from the iit-backend repo and associated Airtable ticket link.

No.

### CODE REVIEW

File: src/components/pages/NotFoundPage.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  return (
+    <div className="page col-12">
+      <div className="center-container">
+        <img src={BrokenAb} height={"30%"} alt="" />
+        <p className="not-found-subheader">{Translate("NotFound", ["PartOne"])}</p>
+        <p>{Translate("NotFound", ["PartTwo"])}</p>
+        <Button 
+          className="not-found-button"
+          onClick={() => history.push(`/${language}/Explore`)}>
```
Comment (annoyingpr) [inline src/components/pages/NotFoundPage.tsx:30]: Just for my understanding - what's the difference between `history.push()` and having a direct link to `/Explore` here?

**Review by annoyingpr — APPROVED** [review]
LGTM 🚀

File: src/utils/translate/en.json [hunk 2]
```diff
@@ -260,6 +260,11 @@
     "PartOne": "No data is available for the filters you've specified.",
     "PartTwo": "Please try a different filter combination."
   },
+  "NotFound": {
+    "PartOne": "Page Not Found",
+    "PartTwo": "Uh oh, we couldn't find the page you're looking for! Please try another URL or visit our Explore page to get back on track.",
```
Comment (brett-stephen) [inline src/utils/translate/en.json:265]: >     get back on track."

10/10 pun if intentional

**Review by brett-stephen — APPROVED** [review]
looks good.

File: src/utils/translate/en.json (hunk 2 above)
Comment (atatmaja) [reply src/utils/translate/en.json:265]: lmaoooo glad someone caught on!!!!

File: src/components/pages/NotFoundPage.tsx (hunk 1 above)
Comment (atatmaja) [reply src/components/pages/NotFoundPage.tsx:30]: React router dom uses the history object to manage session state (what routes users have visited). The `history.push()` method will allow us to route while updating this session state. It's not super necessary but best practice when using react router dom, and we may want to use features provided by the history object in the future

**Pull request merged**